pkg/event/processor_test.go

```diff
@@ -506,7 +504,7 @@ func BenchmarkWithQueueSize(b *testing.B) {
 	for _, merge := range merges {
 		var totalSent = 0
 		var numberRun = 0
-		b.Run(merge.name, func (b *testing.B) {
+		b.Run(merge.name, func(b *testing.B) {
```

Comment:
I think this should've been caught by our linters. Can we update them to catch these in a follow-up PR?

Reply:
this is IDE's thing,  there is difference between Goland and VSC, I will look into that